pkgs/tools/security/kiterunner/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "kiterunner";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "assetnote";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "084jywgqjj2hpaprdcb9a7i8hihphnfil0sx3wrlvjpa8sk0z1mw";
+  };
+
+  vendorSha256 = "1nczzzsnh38qi949ki5268y39ggkwncanc1pv7727qpwllzl62vy";
+
+  preBuild = ''
+    buildFlagsArray+=("-ldflags" "-s -w -X github.com/assetnote/kiterunner/cmd/kiterunner/cmd.Version=${version}")
+  '';
```

Comment:
```suggestion
  buildFlagsArray = [ "-ldflags=-s -w -X github.com/assetnote/kiterunner/cmd/kiterunner/cmd.Version=${version}" ];
```

Reply:
This will change with https://github.com/NixOS/nixpkgs/pull/125520. One too many issue with `buildFlagsArray = []` in the past.